concent_api/core/transfer_operations.py

```diff
@@ -107,95 +107,92 @@ def store_pending_message(
 
 
 def create_file_transfer_token_for_concent(
-    report_computed_task: message.tasks.ReportComputedTask,
-    operation: FileTransferToken.Operation,
-    should_add_source: bool = False,
+    subtask_id: str = None,
+    source_package_path: Optional[str] = None,
+    source_size: Optional[int] = None,
+    source_package_hash: Optional[str] = None,
+    result_package_path: str = None,
```

Comment:
Still few `Optional` missing.

Reply:
I have checked and deleted default None value. Done